fvdi/modules/ft2/ft2.c

```diff
@@ -17,11 +17,13 @@
 #include FT_GLYPH_H
 #include FT_SYNTHESIS_H
 #include FT_STROKER_H
+#include FT_FONT_FORMATS_H
 #else
 #include <freetype/freetype.h>
 #include <freetype/ftglyph.h>
 #include <freetype/ftsynth.h>	/* FT_GlyphSlot_Embolden */
 #include <freetype/ftstroke.h>	/* FT_Stroker, ... */
+#include <freetype/fnfntfmt.h>
```

Comment:
This is [documented as needed](https://freetype.org/freetype2/docs/reference/ft2-font_formats.html) by FT_Get_Font_Format, but this block of includes never seems to get used.